src/lib/sidebar/PipelineSchemeForm.svelte

```diff
@@ -10,15 +10,18 @@
     TextArea,
     TextInput,
   } from "lib/govuk";
-  import { gjScheme } from "stores";
+  import { gjSchemeCollection } from "stores";
   import ATF4Type from "../forms/ATF4Type.svelte";
+  import { getFirstSchemeOrEmptyScheme } from "./scheme_data";
 
   let showModal = false;
+  let scheme = getFirstSchemeOrEmptyScheme($gjSchemeCollection);
```

Comment:
A TODO here would be good to mark we'll change this in the next step, which'll add UI support for multiple schemes. Right now we're assuming there's 0 or 1 schemes filled out